src/test/java/uk/gov/hmcts/reform/iacasepaymentsapi/infrastructure/security/oauth2/IdamAuthoritiesConverterTest.java

```diff
@@ -87,6 +87,33 @@ public void should_return_correct_granted_authority_collection() {
         assertEquals(expectedGrantedAuthorities, grantedAuthorities);
     }
 
+    @Test
+    public void should_return_user_email() {
```

Comment:
we do not need to change Authorities converter

Reply:
Removed